README.md

```diff
@@ -71,6 +71,7 @@ The core contributors so far are:
  * [Will Montgomery](https://github.com/csdigi)
  * [Fabian Siddiqi](https://github.com/FS89)
  * [Diana-Maria Costea](https://github.com/dianacostea)
+ * [Dimitar Pavlov](https://github.com/dimpavloff)
```

Comment:
We did just talk about removing this list (as not many people remember to update it / are unsure when they should). Let's see what pebers thinks..